src/js/createCase.js

```diff
@@ -7,7 +7,7 @@ import { spinUpStore } from './redux-config';
 // Lit of products that are bundles
 const BUNDLE_PRODUCTS = [
   { id: 'openshift', name: 'Red Hat OpenShift Cluster Manager' },
-  { id: 'insights', name: 'Red Hat Insights' },
+  { id: 'insights', name: 'Insights' },
```

Comment:
None of this file needs to change. These fields are mapped to PCM's fields

Reply:
Good point! I'll add it back.